Source/Core/DolphinQt/MenuBar.cpp

```diff
@@ -106,6 +107,9 @@ void MenuBar::OnEmulationStateChanged(Core::State state)
     m_recording_stop->setEnabled(false);
   m_recording_play->setEnabled(!running);
 
+  // Options
+  m_controllers_action->setEnabled(running ? !NetPlay::IsNetPlayRunning() : true);
```

Comment:
This is not what I originally suggested, and it loses the semantics of Netplay being what changes the behavior.

Reply:
I still don't agree, and believe it's just as readable in either this or your suggested form, but I'm not going to argue over the swapping of a ternary operation.

Reply:
I prefer the final version, and here's my reasoning:

To a reader seeing this code the first time, `IsNetPlayRunning()` effectively translates into "is weird stuff happening right now?". If I know I don't care about netplay, I can skip to the false branch and see the control's always enabled. Because it's the more explanatory condition, it should be the one placed as the "condition" even though this is really just a boolean logic statement.